content/en/docs/about/overview.md

```diff
@@ -9,43 +9,71 @@ weight: 1
 
 ![Sigstore](sigstore-logo_horizontal-color.svg)
 
-**Sigstore empowers software developers to securely sign software artifacts such as release files, container images, binaries, bill of material manifests and more. Signing materials are then stored in a tamper-resistant public log.**
+**Sigstore empowers software developers and consumers to securely sign and verify software artifacts such as release files, container images, binaries, software bills of materials (SBOMs), and more. The signing materials are stored in a tamper-resistant public log so there’s no need to manage or store keys.**
```

Comment:
We shouldn't say that signing materials or keys are signed in the log, as logs are only meant to be witnesses to events, not storage. Instead, something like:

```
Signatures are generated with ephemeral signing keys so there's no need to manage keys.
Signing events are recorded in a tamper-resistant public log so software developers can audit signing events.
```